### DISCUSSION TITLE
ambiguity with `!=` and `/=` when `cv24` is enabled

### DISCUSSION THREAD

**Discussion by scarf005** [tonsky/FiraCode#1466] [Ideas]
![image](https://user-images.githubusercontent.com/54838975/189929051-4617f63d-6b01-4316-999b-843b9c9574b8.png)
[`cv24`]([url](https://github.com/tonsky/FiraCode/wiki/How-to-enable-stylistic-sets)) makes `/=` look same as `!=`. while it looks great on haskell project, it might be confusing since those two renders exactly same. here's some approaches:

- change `/=` operator to use different glyph, like that of https://github.com/i-tu/Hasklig ![image](https://user-images.githubusercontent.com/54838975/189930128-ea6d9cc9-fb4f-4ee8-9be7-af522bbd150a.png)
- a way to disable `!=` (cv25?)

**Comment by tonsky** [comment]
Does != mean anything in Haskell?

**Reply by scarf005** [reply]
![image](https://user-images.githubusercontent.com/54838975/190270290-2648321f-536d-45bd-82d1-de0803a838ee.png)
[hoogle search result](https://hoogle.haskell.org/?hoogle=!%3D) 

not in standard library, however some external libraries define them as it's a valid function name in Haskell
